packages/web3/artifacts/contracts/foobar/contract.sol/Foobar.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
abi json files should be included in repository?

Reply:
its imported for the [wagmi.config.ts](https://github.com/wakeuplabs-io/react-monorepo-template/blob/5b88114b50e1a82fb87fc1834464ff673397f5d5/packages/web3/wagmi.config.ts)